actions/wit/entrypoint.sh

```diff
@@ -5,7 +5,14 @@ set -euxo pipefail
 : ${INPUT_COMMAND:?'The `command` input argument must be set'}
 
 if [[ -n "$INPUT_FORCE_GITHUB_HTTPS" ]]; then
-  git config --global url.'https://github.com/'.insteadOf 'git@github.com:'
+  if [[ -n "$INPUT_TOKEN" ]]; then
+    # Don't echo the token
+    set +x
+    git config --global url."https://x-access-token:${INPUT_TOKEN}@github.com/".insteadOf 'git@github.com:'
```

Comment:
I realize you didn't introduce this usage of `--global`, but I would like to make the comment that this command would change the behavior of git commands unrelated to the current workspace

Reply:
I guess the question is: Does removing `--global` break anything about the intended usage of the action?

Reply:
[The `git-config` documentation](https://git-scm.com/docs/git-config#Documentation/git-config.txt---global) seems to say that without `--global`, the changes only apply to the local repository's `.git/config`, which doesn't actually work here because we want these changes to apply to every repository in the workspace.

Reply:
Since these commands are being run within the context of the wit container, I think the config file being modified is the one inside the container. Therefore the scope actually is limited to the workspace because the container only persists for the duration of this GitHub action.

Reply:
Oh I misread the whole diff, I didn't realize this was in a github action